DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -153,41 +153,20 @@ namespace cms {
       return dd4hep::isA<Shape>(solid());
     }
 
-    template <typename... Ts>
-    auto isABox(Ts&&... ts) const -> decltype(isA<dd4hep::Box>(std::forward<Ts>(ts)...)) {
-      return isA<dd4hep::Box>(std::forward<Ts>(ts)...);
-    }
+    bool isABox() const { return isA<dd4hep::Box>(); }
```

Comment:
@ghugo83 - I agree with you here. There is no need to scare people with variadic templates and perfect forwarding :-)

However, it got me thinking that logically  _**"a `View` is not a `Box`"**_, but its `shape` is. These methods implement legacy API. Digging a bit more and it looks like there is not much need for them, not really. Half of them are not used and some that would be used are not implemented. The latter cases use the `dd4hep` API directly.  I'd suggest to remove these methods entirely from the `FilteredView`. 

I'd suggest:
```c++
dd4hep::isA<dd4hep::Polycone>(fv.solid());
```
instead of:
```c++
fv.isA<dd4hep::Polycone>();
```

Reply:
Ok @ianna , will replace `fv.isA<dd4hep::Polycone>()` by `dd4hep::isA<dd4hep::Polycone>(fv.solid())` to directly use the dd4hep interface.

Reply:
@ianna Haha yes I found it more elegant.